diff -urN udpcast-20060929-orig/Makefile.in udpcast-20060929/Makefile.in
--- udpcast-20060929-orig/Makefile.in	2006-09-17 20:25:46.000000000 +0200
+++ udpcast-20060929/Makefile.in	2006-10-29 10:25:39.757683120 +0100
@@ -2,6 +2,7 @@
 srcdir=@srcdir@
 VPATH=@srcdir@
 
+DESTDIR		= ""
 prefix      = @prefix@
 exec_prefix = @exec_prefix@
 bindir      = @bindir@
@@ -53,10 +54,10 @@
 	udp-sender.1 udp-receiver.1 *~
 
 install: udp-sender udp-receiver
-	install -d $(prefix)/sbin
-	install -m 755 udp-sender udp-receiver $(prefix)/sbin
-	install -d $(mandir)/man1
-	install -m 644 udp-sender.1 udp-receiver.1 $(mandir)/man1
+	install -d $(DESTDIR)$(prefix)/sbin
+	install -m 755 udp-sender udp-receiver $(DESTDIR)$(prefix)/sbin
+	install -d $(DESTDIR)$(mandir)/man1
+	install -m 644 udp-sender.1 udp-receiver.1 $(DESTDIR)$(mandir)/man1
 
 udp-sender$(EXESUFFIX): udp-sender.o socklib.o udpcast.o rate-limit.o \
 	sender-diskio.o senddata.o udps-negotiate.o \
