--- udpcast-20060619/Makefile.orig	2005-12-23 10:55:04.000000000 +0000
+++ udpcast-20060619/Makefile	2006-06-25 15:11:41.615997000 +0000
@@ -1,14 +1,15 @@
 DESTDIR =
 
 #CFLAGS = -O4 -Wall -DNDEBUG -DLOSSTEST
-CFLAGS =-Wall -Wshadow -DBB_FEATURE_UDPCAST_FEC -D_FILE_OFFSET_BITS=64 -DUSE_SYSLOG -DUSE_ASSEMBLER -O6
-LDFLAGS =-s -Wl,-warn-common
+override CFLAGS +=-Wall -Wshadow -DBB_FEATURE_UDPCAST_FEC -D_FILE_OFFSET_BITS=64 -DUSE_SYSLOG
+override LDFLAGS += -Wl,-warn-common
+
 LIBS=-lpthread
 BUSYBOX=../udp-busybox/busybox
 BBTARGET=$(BUSYBOX)/udpcast
 EXESUFFIX=
 
-CFLAGS += -DNO_BB
+override CFLAGS += -DNO_BB
 
 all: udp-receiver$(EXESUFFIX) udp-sender$(EXESUFFIX) \
 	udp-receiver.1 udp-sender.1
