diff -urN udpcast-20081213/Makefile.in.orig udpcast-20081213/Makefile.in
--- udpcast-20081213/Makefile.in.orig	2008-11-16 17:36:25.000000000 +0000
+++ udpcast-20081213/Makefile.in	2009-03-03 12:58:40.496625476 +0000
@@ -30,7 +30,7 @@
 
 # For additional warnings & checks, add -Wstrict-prototypes -Wshadow to the
 # following:
-CFLAGS +=-Wall -DBB_FEATURE_UDPCAST_FEC -D_FILE_OFFSET_BITS=64 -DUSE_SYSLOG -DUSE_ASSEMBLER -O6
+CFLAGS +=-Wall -DBB_FEATURE_UDPCAST_FEC -D_FILE_OFFSET_BITS=64 -DUSE_SYSLOG -DUSE_ASSEMBLER
 LDFLAGS +=-s
 LIBS +=@LIBS@
 
