--- u/Makefile.in.orig	2007-02-08 20:39:28.416109157 +0000
+++ u/Makefile.in	2007-02-08 20:54:05.846828596 +0000
@@ -30,7 +30,7 @@
 
 
 #override CFLAGS += -O4 -Wall -DNDEBUG -DLOSSTEST
-override CFLAGS +=-Wall -Wstrict-prototypes -Wdeclaration-after-statement -Wshadow -DBB_FEATURE_UDPCAST_FEC -D_FILE_OFFSET_BITS=64 -DUSE_SYSLOG -DUSE_ASSEMBLER -O6
+override CFLAGS +=-Wall -Wstrict-prototypes -Wdeclaration-after-statement -Wshadow -DBB_FEATURE_UDPCAST_FEC -D_FILE_OFFSET_BITS=64 -DUSE_SYSLOG -DUSE_ASSEMBLER
 override LDFLAGS +=-s
 LIBS +=@LIBS@
 
 
