diff -Nur udpcast-20071228.org/configure.in udpcast-20071228/configure.in
--- udpcast-20071228.org/configure.in	2007-06-01 09:07:02.000000000 +0000
+++ udpcast-20071228/configure.in	2008-06-29 13:58:32.923581500 +0000
@@ -188,7 +188,7 @@
 objs=`echo $srcdir/*.c | sed 's/\.c$/.o/' `
 if [ "X$GCC" = "Xyes" ] ; then
     if [ "$host_cpu" = i486 ] ; then
-	    CFLAGS="$CFLAGS -m486"
+	    CFLAGS="$CFLAGS"
     fi
     Wall=-Wall
     if [ "$host_os3" = sunos ] ; then
