diff -urN udpcast-20071228.orig/Makefile.in udpcast-20071228/Makefile.in
--- udpcast-20071228.orig/Makefile.in	2007-12-26 23:14:27.000000000 +0100
+++ udpcast-20071228/Makefile.in	2008-02-02 16:29:57.211474115 +0100
@@ -30,7 +30,7 @@
 
 
 #override CFLAGS += -O4 -Wall -DNDEBUG -DLOSSTEST
-override CFLAGS +=-Wall -Wstrict-prototypes -Wshadow -DBB_FEATURE_UDPCAST_FEC -D_FILE_OFFSET_BITS=64 -DUSE_SYSLOG -DUSE_ASSEMBLER -O6
+override CFLAGS +=-Wall -Wstrict-prototypes -Wshadow -DBB_FEATURE_UDPCAST_FEC -D_FILE_OFFSET_BITS=64 -DUSE_SYSLOG -DUSE_ASSEMBLER 
 override LDFLAGS +=-s
 LIBS +=@LIBS@
 
